external/loaders/loaders/mappers/_fine_res.py

```diff
@@ -19,6 +20,15 @@ class MLTendencies(Protocol):
     dQ2: xr.DataArray
 
 
+class NudgedRun(Protocol):
+    tendency_of_air_temperature_due_to_dynamics: xr.Dataset
```

Comment:
@oliverwm1 Can you confirm that these are the names in your prescribed run outputs? Thanks.

Reply:
Yes these are the names.